### ISSUE TITLE
have the support of NVIDIA RTX 40X GPU?

### ISSUE THREAD

**Issue by kafei123456** [Qsingle/MegEngine_CU11#3]

**Comment by Qsingle** [comment]
Yeah, it also supports the RTX 40X GPU, you can download the latest version 1.13.0 for the Windows system through the [Baidu Cloud Driver](https://pan.baidu.com/s/1qHkv9slHvVSNvSfbjkXgWg&pwd=bwwn).
